test/ctia/task/migration/migrate_es_stores_test.clj

```diff
@@ -577,7 +577,13 @@
   (let [{:keys [get-in-config]} (helpers/get-service-map app :ConfigService)
         {:keys [all-stores]} (helpers/get-service-map app :StoreService)
         services (app->MigrationStoreServices app)
-        conn (es-conn get-in-config)]
+        conn (es-conn get-in-config)
+        migrated-store-keys [:incident
+                             :asset
+                             :attack-pattern
+                             :judgement
+                             :sighting
+                             :malware]]
```

Comment:
migrate less indices (still validate the process), that's workaround for introduced shard limit on ES7
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/breaking-changes-7.0.html#_cluster_wide_shard_soft_limit